test/e2e/network_test.go

```diff
@@ -869,6 +933,16 @@ func (tc *testContext) createWindowsServerDeployment(name string, command []stri
 								},
 							},
 							VolumeMounts: volumeMounts,
+							Resources: v1.ResourceRequirements{
+								Limits: v1.ResourceList{
+									v1.ResourceCPU:    resource.MustParse("500m"),
+									v1.ResourceMemory: resource.MustParse("500m"),
+								},
```

Comment:
Shouldn't memory be measured in Mi (Megabytes) or Gi (gigabytes)?
Are you sure about these limits, is there any recommendation we are following here?

Reply:
Good catch, updated.

Also updated the commit  message with information about the provided values